umm-spec-lib/src/cmr/umm_spec/test/umm_record_sanitizer.clj

```diff
@@ -160,9 +162,7 @@
       (sanitize-umm-record-contacts :ContactPersons)
       (sanitize-umm-record-contacts :ContactGroups)
       (sanitize-umm-record-related-url :CollectionCitations)
-      (sanitize-umm-record-related-url :PublicationReferences)
-      (update-in-each [:PublicationReferences] sanitize-online-resource)
-      (update-in-each [:CollectionCitations] sanitize-online-resource)))
+      (sanitize-umm-record-related-url :PublicationReferences)))
```

Comment:
I see you removed sanitize-online-resource for PublicationReferences and CollectionCitation in sanitize-umm-record-urls, then add them back later in sanitized-umm-c-record which calls sanitize-umm-record-urls. That's good because I was wondering why you need to change sanitized-umm-c-record with the service schema update...  
The question is:  Did you do this to make the code look better or sanitize-umm-record-urls can't contain the sanitize-online-record for sanitized-umm-s-record to work? I'm guessing you want to categorize it better but leaving sanitize-online-record there won't hurt? so the real change is only on ContactInformation and umm-s-record sanitization only?